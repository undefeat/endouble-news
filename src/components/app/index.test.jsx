import React from 'react';
import shallow from 'react-test-renderer/shallow';
import App from '.';

describe('App', () => {
    it('should correctly render', () => {
        const tree = shallow.createRenderer().render(<App />);

        expect(tree).toMatchSnapshot();
    });
});
