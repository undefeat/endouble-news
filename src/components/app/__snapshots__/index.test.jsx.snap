// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`App should correctly render 1`] = `
<React.Fragment>
  <div
    className="app"
  >
    <header
      className="app__header"
    >
      <FilterPanel
        categories={
          Array [
            "business",
            "entertainment",
            "general",
            "health",
            "science",
            "sports",
            "technology",
          ]
        }
        countries={Array []}
        onCategoryChanged={[Function]}
        onCountryChanged={[Function]}
        onSearchPhraseChanged={[Function]}
        q=""
        selectedCategory="general"
        selectedCountry="us"
      />
      <SortPanel
        onSortByChanged={[Function]}
        sortBy="date"
        sortOptions={
          Array [
            "date",
            "source",
          ]
        }
      />
    </header>
    <ArticleList
      articles={Array []}
      fetching={false}
      onPageChanged={[Function]}
      page={1}
      pageSize={12}
      totalResults={0}
    />
    <Footer />
  </div>
  <BackToTop />
</React.Fragment>
`;
